src/main/webapp/resources/js/components/AnalysesTable/AnalysesTable.jsx

```diff
@@ -25,11 +25,13 @@ const UPDATE_PROGRESS_DELAY = 60000;
 
 /**
  * Displays the Analyses Table for both user and admin pages.
+ * @param {boolean} canManage - If the current user can manage
+ * the project or not. Defaults to the window.PAGE variable if
+ * one is not passed in.
  * @returns {*}
  * @constructor
  */
-export function AnalysesTable() {
-  const CAN_MANAGE = window.PAGE.canManage;
+export function AnalysesTable({ canManage = window.PAGE.canManage }) {
```

Comment:
Get this from using redux-toolkit and the projectApi

Reply:
As discussed we couldn't use the projectApi in here. Updated the ProjectAnalysesPage to use the projectApi and the AnalysesPage to use the page variables.

Reply:
Updated in e28ce1f99099d12afd4ef7c9b50dae8c7bc964fb and 75b253d54e9481f6b6a82b59fc7137ad4b9523c0